tests/driver_pulse_counter/main.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2017 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the PULSE_COUNTER driver
+ *
+ * @author      Hyung-Sin Kim <hs.kim@cs.berkeley.edu>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "xtimer.h"
+#include "pulse_counter_params.h"
+
+#define SLEEP       US_PER_SEC
+
+int main(void)
+{
+    pulse_counter_t dev;
+    int16_t count;
```

Comment:
why a signed value ? I thought a count value is supposed to be positive.

Reply:
pulse counter driver returns pulse count value as the int16_t type for SAUL compatibility.